docs/source/models/AOFDTD.rst

```diff
@@ -54,11 +54,13 @@ between positions where the field quantities are known.
 
 The above discretization uses one neighbor to each side from the point where the derivative is calculated yielding a
 second order accurate approximation of the derivative.
-Using more neighbors for the approximation of the spatial derivative is possible in PIConGPU and reduces the
-discretization error.
-Which is to say that the order of the method is increased.
-The error order scales with twice the number of neighbors :math:`M` used to approximate the derivative.
-The arbitrary order finite difference of order :math:`2M` reads
+Using more neighbors for finite difference calculation of the spatial derivatives is possible in PIConGPU and increases the approximation order of these derivatives.
+Note, however, that the order of the whole Maxwell's solver also depends on accuracy of :math:`\vec{J}` calculation on the grid.
+For those values PIConGPU provides only second-order accuracy in terms of time and spatial grid steps (as the underlying discretized continuity equation is of that order) regardless of the chosen field solver.
+Thus, in the general case the arbitrary order finite difference solver as a whole still has second order accuracy in space, and only provides arbitrary order in finite difference approximation of curls.
```

Comment:
I suggest replacing 'arbitrary order finite difference' solver at the beginning of the sentence by 'Maxwell solver', as you have used it in the discussion.

Reply:
Good point, will do.